pydantic/types.py

```diff
@@ -2822,3 +2822,17 @@ def __get_pydantic_core_schema__(cls, source_type: Any, handler: GetCoreSchemaHa
 That is, instead of a [`ValidationError`][pydantic_core.ValidationError] being propagated up and the entire iterable being discarded
 any invalid items are discarded and the valid ones are returned.
 """
+
+
+# Can't use `Field(deprecated=True)` because of a circular import
+Deprecated = Annotated[T, deprecated('')]
```

Comment:
I think given I'd like to require a string message instead of a boolean value when using the `deprecated` keyword argument to `Field`, we should drop this type and just require people to manually add a deprecation annotation if they want to deprecate a type like this.

Do you object to that?

Reply:
Makes sense, it would be nice to have this helper type still around to have the following at runtime:

`my_field: Deprecated[int]  # accessing my_field would raise a runtime warning with a default templated message, e.g. "my_field is deprecated"`.

But considering people usually want to provide an alternative solution to the deprecated field, it wouldn't be that useful.